util/log.go

```diff
@@ -3,23 +3,21 @@ package util
 import (
 	"os"
 
-	logging "github.com/ipfs/go-ipfs/Godeps/_workspace/src/github.com/whyrusleeping/go-logging"
+	"github.com/ipfs/go-ipfs/Godeps/_workspace/src/github.com/Sirupsen/logrus"
```

Comment:
keep go-logging is euphemism for kill logrus (as this will be the course of action written in the commit message, not the former)